internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -45,26 +45,40 @@ const (
 	maxIndexNameLength = 63
 )
 
-// metadata is a type to structure methods that work with metadata storing and getting.
-//
-// Metadata consists of collections and indexes settings.
-type metadata struct {
+// metadataStorage offers methods to store and get metadata for the given database and collection.
+type metadataStorage struct {
 	tx         pgx.Tx
 	db         string
 	collection string
 }
 
-// newMetadata returns a new instance of metadata for the given transaction, database and collection names.
-func newMetadata(tx pgx.Tx, db, collection string) *metadata {
-	return &metadata{
+// metadata stores information about FerretDB collections and indexes.
+type metadata struct {
+	pgtable string          // Corresponding PostgreSQL table name
```

Comment:
```suggestion
	table string
```

No other types of tables are possible in `pgdb`, we consistently use "table" to refer to PostgreSQL table (as opposed to MongoDB/FerretDB collection), and we don't add comments to obvious fields.

Please do the same everywhere.

Reply:
Then the naming is inconsistent: index in pg is called `pgindex`, but table in pg is called just `table`.

Reply:
I thought that struct described our document, not PostgreSQL internals?

Reply:
No, this struct describes PostgreSQL-related things such as table name.

Reply:
Yes. But should we use exactly the same names as PostgreSQL internals? Or we could use names that we use in the BSON/JSON document, to remove one extra abstraction?

Reply:
fixed

Reply:
> and we don't add comments to obvious fields

That part was missed. I think fields in `metadata` and `metadataIndex` are self-describing

Reply:
fixed